contracts/contracts/open_gsn/UmbraPaymaster.sol

```diff
@@ -26,7 +26,7 @@ contract UmbraPaymaster is BasePaymaster {
     (signature, approvalData, maxPossibleGas); // to silence compiler warnings
 
     require(relayRequest.request.to == umbraAddr, "UmbraPaymaster: Not Target");
-    return (abi.encode(0x0), true);
+    return ("", true);
```

Comment:
What's the rationale behind this change?

Reply:
I just pulled this from their updated Paymaster reference implementation for the mainnet v2.0. Not sure it actually matters.